conversations.yaml

```diff
@@ -0,0 +1,536 @@
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description:
+            Returns the list of available conversations.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConversationsList'
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+        500:
+          description: Internal server error
+      security:
+        - conversationsOauth: []
+components:
+  schemas:
+    ConversationsList:
+      properties:
+        items:
+          type: array
+          items:
+            properties:
+              guid:
+                type: string
+                format: uuid
+              title:
+                type: string
+              description:
+                type: string
+              keyword:
+                type: array
+                items:
+                    type: string
+              labels:
+                type: array
+                items:
+                    type: string
+              created_at:
+                type: string
+                format: date-time
+                description: "The date of creation in ISO-8601 UTC format (ex: yyyy-mm-ddThh:mm:ssZ)."
+              updated_at:
+                type: string
+                format: date-time
+                description: "The last modification date in ISO-8601 UTC format (ex: yyyy-mm-ddThh:mm:ssZ)."
+        _total:
+          type: integer
+          format: int64
+    CustomerProfileData:
+      properties:
+        first_name:
+          type: string
+        last_name:
+          type: string
+        profile_pic:
+          type: string
+          description: a publicly accessible image URL.
+        locale:
+          type: string
+          description: the ISO-639 or ISO-3166 country code.
+        timezone:
+          type: integer
+          description: "Time zone offset, the number relative to GMT, (ex: -8)."
+        phone_number:
+          type: string
+          description: must be an E.164 format number.
+    OutboundMessageRequest:
+      properties:
+        recipient:
+          properties:
+            id:
+              type: string
+              description:
+                An identifier unique to the customer, such as phone number,
+                email, or another channel-specific value. For phone numbers,
+                the identifier must be in E.164 format.
+        message:
+          properties:
+            text:
+              type: string
+            quick_replies:
+              type: array
+              items:
+                properties:
+                  content_type:
+                    type: string
+                  title:
+                    type: string
+                  payload:
+                    type: string
+                  image_url:
+                    type: string
+                    format: uri
```

Comment:
It may be useful to add `description:` fields to these to indicate what they do in the larger Conversation display?  I.e. how should the channel adapter think about rendering these?